src/app/portlets/dot-edit-page/layout/dot-edit-layout-grid/dot-edit-layout-grid.component.ts

```diff
@@ -66,35 +52,21 @@ export class DotEditLayoutGridComponent implements OnInit {
 
     constructor(
         public messageService: MessageService,
-        private dotConfirmationService: DotConfirmationService,
-        private dotEditLayoutGridService: DotEditLayoutGridService
+        private dotConfirmationService: DotConfirmationService
     ) {}
 
     ngOnInit() {
         this.messageService.getMessages(this.i18nKeys).subscribe();
-        if (this.pageView) {
```

Comment:
It's important to understand that in the majority of the the cases, component will not do much, they should **almost** all the time to receive data from an attribute and render out, respond to events and that's it.